ui/src/components/content/home/Home.tsx

```diff
@@ -6,7 +6,11 @@
 import WelcomeCard from './WelcomeCard';
 
 function Home(): JSX.Element {
-    return <WelcomeCard />;
+    return (
+        <div className='home-data-display'>
```

Comment:
is it possible to directly pass the class with `className` prop to WelcomeCard component instead of additional div?

Reply:
div is added to make the layout of the page uniform, maybe the registr panel page button position needs to be adjusted.